docs/pages/deploy-a-cluster/helm-deployments/azure.mdx

```diff
@@ -0,0 +1,325 @@
+```
+
+Teleport will store session recordings in a Blob Storage account. Optionally,
+access can be restricted to just the AKS outbound address, or the account can be
+made part of the virtual network that the AKS cluster is using.
+
+```code
+$ az storage account create --resource-group <Var name="teleport-rg" /> --name <Var name="teleport-blob" /> \
+    --allow-blob-public-access false
+$ az role assignment create --role "Storage Blob Data Owner" --assignee-principal-type ServicePrincipal \
+    --assignee-object-id "$(az identity show --resource-group <Var name="teleport-rg" /> --name teleport-id --query principalId -o tsv)" \
+    --scope "$(az storage account show --resource-group <Var name="teleport-rg" /> --name <Var name="teleportblob" /> --query id -o tsv)""
+```
+
+We'll use Workload Identity to authenticate to those services, so we'll add
+federated credentials for the Teleport service account used by the Auth Service.
+
+```code
+$ az identity federated-credential create --resource-group <Var name="teleport-rg" /> --identity-name teleport-id \
+    --name aks --audience api://AzureADTokenExchange \
+    --subject system:serviceaccount:<Var name="teleport-ns" />:<Var name="teleport-release" /> \
+    --issuer "$(az aks show --resource-group <Var name="aks-rg" /> --name <Var name="aks-name" /> --query oidcIssuerProfile.issuerUrl -o tsv)""
+```
+
+## Step 3/5. Set values to configure the cluster
+
+<Details title="License Secret" scopeOnly={false} scope={"enterprise"} opened={true}>
+
+Before you can install Teleport Enterprise in your Kubernetes cluster, you will need to
+create a secret that contains your Teleport license information.
+
+(!docs/pages/includes/enterprise/obtainlicense.mdx!)
+
+Create a secret from your license file. Teleport will automatically discover
+this secret as long as your file is named `license.pem`.
+
+```code
+$ kubectl create namespace <Var name="teleport-ns" />
+$ kubectl -n <Var name="teleport-ns" /> create secret generic license --from-file=license.pem
+```
+
+</Details>
+
+Now we'll configure the `teleport-cluster` Helm chart to use the `azure` mode. Create a
+file called `azure-values.yaml` containing the values you've selected above:
+
+<ScopedBlock scope={["oss"]}>
+
+```code
+$ cat >azure-values.yaml <<EOF
+chartMode: azure
+# Name of your cluster. Use the FQDN you intend to configure in DNS later
+clusterName: teleport.example.com
+azure:
+  databaseHost: <Var name="teleport-pg" />.postgres.database.azure.com
+  databaseUser: <Var name="pguser" />
+  sessionRecordingStorageAccount: <Var name="teleportblob" />.blob.core.windows.net
+  # Whether to mirror audit log entries to stdout in JSON format (useful for external log collectors)
+  auditLogMirrorOnStdout: false
+  clientID: $(az identity show --resource-group <Var name="teleport-rg" /> --name teleport-id --query clientId -o tsv)
```

Comment:
It's probably providing people with a way to get and then insert this themselves, as many people who read these guides don't understand what a heredoc is or how it works.

Reply:
Done in 11af5ab